### DISCUSSION TITLE
Omi-chan's blog?

### DISCUSSION THREAD

**Discussion by liftedpixel** [omigroup/omi-chan#26] [Ideas]
![traits](https://user-images.githubusercontent.com/727189/155852417-347e3880-912c-42f4-9a14-93b8ef26e3d1.png)
User omitted, but let me know if you want me to put your name!

Highlighting this idea got me thinking 'how' we integrate Omi-chan into the website.
A blog 'run' by Omi-chan about various avatar things might be a good vector for info/ideas like this?

A post about interop in human speak, encouraging readers to consider what traits are part of their virtual identity, and maybe make it KNOWN so creators know what kind of assets to make.

What is an avatar? type posts for non VR types with nice info-graphs using the Omi-chan models, and even in 3D with that Wordpress plugin available: https://github.com/antpb/three-object-viewer

I feel I could make a giant list of possible posts for a blog like this.
Please add your thoughts, Omi-chan blog post ideas, etc!

**Comment by LightLodges** [comment]
I'd love to see a number of people videoblogging with omichan (and OMICAT!) - some of this content is already written and we have a team of producers working within OMI who can help write/produce. We tried a few test shots to see what this can look like in Animaze with the early avatar file tests (see the discord thread for a mix of test shots as grey omichan and generic omicat). For Streaming shows and interviews we have a Streamyard pro account for this team that can be used/reused. Animaze allows for multiple avatars to be hotkeyed for easy switching in a broadcast or videoblog out.
